vms/platformvm/state/state.go

```diff
@@ -834,9 +898,14 @@ func (s *state) GetSubnetOwner(subnetID ids.ID) (fx.Owner, error) {
 		return nil, fmt.Errorf("%q %w", subnetID, errIsNotSubnet)
 	}
 
+	s.SetSubnetOwner(subnetID, subnet.Owner)
 	return subnet.Owner, nil
 }
 
+func (s *state) SetSubnetOwner(subnetID ids.ID, owner fx.Owner) {
```

Comment:
Can we get a UT on this? `GetSubnetOwner` is having its behavior changed and CI is passing so it looks like we're missing some coverage on `GetSubnetOwner`